src/cockpit/389-console/src/lib/schema/schemaModals.jsx

```diff
@@ -411,60 +395,62 @@ class AttributeTypeModal extends React.Component {
                     onClose={closeModal}
                     actions={btnList}
                 >
-                    <Form horizontal>
-                        <FormGroup controlId="atName">
-                            <Col sm={4}>
-                                <ControlLabel title="Name of the attribute">
-                                    Attribute Name
-                                </ControlLabel>
-                            </Col>
-                            <Col sm={8}>
-                                <FormControl
-                                    id="atName"
-                                    type="text"
+                    <Form isHorizontal autoComplete="off">
+                        <Grid title="Name of the attribute">
+                            <GridItem className="ds-label" span={3}>
+                                Attribute Name
```

Comment:
I'm trying to add an attribute.
There are two mandatory fields 'Name' and 'Description'. I set both to 'test'.

After I click 'Save' I get an error (which is described in details in the Console log, but actual error notification is not descriptive).

I think we can make the error more verbose so the user knows what's wrong.

Also, I am not sure what is wrong with the attribute I've tried to create. Maybe it's an error in our tools...
Please, check the screenshot (error is yellow in logs)

![image](https://user-images.githubusercontent.com/5634892/128046423-fb3c2709-f6dc-4f92-a70a-a75d8d4081a6.png)

Reply:
Fixed, and fixed a bug, "top" is not an attribute

Reply:
If I first create an attribute (with only Name and Description), then when I try to edit and set Parent Attribute - the browser crashes with 'Uncaught TypeError: s.startsWith is not a function' error